src/Altinn.App.Core/Infrastructure/Clients/Email/EmailNotificationClient.cs

```diff
@@ -0,0 +1,124 @@
+using Altinn.App.Core.Configuration;
+using Altinn.App.Core.Internal.App;
+using Altinn.App.Core.Internal.Email;
+using Altinn.App.Core.Models.Email;
+using Altinn.Common.AccessTokenClient.Services;
+using Microsoft.ApplicationInsights;
+using Microsoft.Extensions.Options;
+using Prometheus;
+using System.Diagnostics;
+using System.Net.Http.Headers;
+using System.Text.Json;
+
+namespace Altinn.App.Core.Infrastructure.Clients.Email;
+/// <summary>
+/// Implementation of the <see cref="IEmailNotificationClient"/> interface using a HttpClient to send
+/// requests to the Email Notification service.
+/// </summary>
+public sealed class EmailNotificationClient : IEmailNotificationClient
+{
+    private readonly IHttpClientFactory _httpClientFactory;
+    private readonly IAppMetadata _appMetadata;
+    private readonly PlatformSettings _platformSettings;
+    private readonly IAccessTokenGenerator _accessTokenGenerator;
+    private readonly TelemetryClient _telemetryClient;
+    private static readonly Counter _orderCount = Metrics
+        .CreateCounter("altinn_app_notification_order_request_count", "Number of notification order requests.", labelNames: ["email"]);
```

Comment:
```suggestion
        .CreateCounter("altinn_app_notification_order_request_count", "Number of notification order requests.", labelNames: ["type", "result"]);
```
Since we want the same label name for both types of notifications